pom.xml

```diff
@@ -328,9 +335,9 @@
               <version>${mapstruct.version}</version>
             </path>
             <path>
-              <groupId>org.hibernate</groupId>
+              <groupId>org.hibernate.orm</groupId>
               <artifactId>hibernate-jpamodelgen</artifactId>
-              <version>5.5.0.Final</version>
+              <version>6.1.6.Final</version>
```

Comment:
hello @humonest-one, you should use <version>${hibernate.version}</version> instead of <version>6.1.6.Final</version>. hibernate.version declared in spring-boot-dependencies-3.0.1.pom